ui/components/app/transaction-activity-empty-state/transaction-activity-empty-state.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
Adding story (ignore incorrect theming for image known issue in storybook)

<img width="614" height="499" alt="Screenshot 2025-09-19 at 2 33 23 PM" src="https://github.com/user-attachments/assets/0137c118-57e3-4454-bc27-1a43f7a0a9fa" />